src/storage.rs

```diff
@@ -46,8 +47,9 @@ impl<T: Storable> Storable for StorableIndex<T> {}
 pub(crate) trait Storable: Serialize + Copy + Debug {}
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
-pub(crate) struct Storage(HashMap<Vec<u8>, Vec<u8>>);
+pub struct Storage(HashMap<Vec<u8>, Vec<u8>>);
```

Comment:
Curious why this needs to be `pub`?

Reply:
I think this was to compile an intermediate state when `Presign` still took `Storage` as input. Probably not necessary any more!